addons/l10n_ua_psbo/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Ukraine - Accounting PSBO',
+    'author': 'ERP Ukraine',
+    'website': 'https://erp.co.ua',
+    'version': '1.5',
+    'description': """
+Ukrainian accounting based on national standards
+================================================
+    """,
+    'category': 'Localization',
+    'depends': ['account', 'l10n_ua'],
+    'data': [
+        'data/account_chart_template.xml',
+        'data/account.account.template.csv',
+        'data/account_tax_template.xml',
+        'data/account_chart_template_config.xml',
+        'data/account_chart_template_data.xml',
+    ],
+    'auto_install': True,
```

Comment:
Could you also add code inside the account/__init__.py file to automatically load the UA chart of accounts on the SaaS.

You can have a look inside the file on how it is done for l10n_de and do the same for l10n_ua
elif country_code == 'DE':
            module_list.append('l10n_de_skr03')
            module_list.append('l10n_de_skr04')

Reply:
ok